### PR TITLE
EVEREST-1618 add affinity

### PR DESCRIPTION
**Pull request by recharte**
Following up the changes introduced in https://github.com/percona/everest-operator/pull/592 we need to add the affinity fields to the everest-operator CRD.

### CODE REVIEW

**Review by oksana-grishchenko — APPROVED** [review]

File: charts/everest/crds/everest.yaml
Comment (mayankshah1607) [reply charts/everest/crds/everest.yaml:522]: > Should we bump the chart version?

That happens automatically when you run the action for publishing the chart

Comment (oksana-grishchenko) [reply charts/everest/crds/everest.yaml:522]: btw, let's add the same rule we have in other repos - block the PR if there are unresolved comments. wdyt?

Comment (mayankshah1607) [reply charts/everest/crds/everest.yaml:522]: How it works is you merge everything to `main`, which is version 0.0.0, and never publish this.

Then you create a new chart release (like 1.4.0), and it updates the chart version and publish the chart

**Review by mayankshah1607 — APPROVED** [review]
LGTM, but if we merge it now, it will become a part of 1.4.0.. maybe we should hold it off until then.

Unlike the CI pipelines in everest and everest-operator, we release directly from main, rather than having a branch like `release-1.4` and cherry-picking changes. Maybe we should change it to be similar like our other pipelines... I'm not sure though, since we don't own this repo 😅

File: charts/everest/crds/everest.yaml
Comment (oksana-grishchenko) [reply charts/everest/crds/everest.yaml:522]: > btw, let's add the same rule we have in other repos - block the PR if there are unresolved comments. wdyt?

oh, we don't own this repo. but it could be beneficial for others too :)

**Comment by recharte** [comment]
> Maybe we should change it to be similar like our other pipelines...

This definitely makes sense to avoid having to hold PRs during the RC stage...

File: charts/everest/crds/everest.yaml
Comment (recharte) [reply charts/everest/crds/everest.yaml:522]: @mayankshah1607 we should bump the chart version, right?

Comment (mayankshah1607) [reply charts/everest/crds/everest.yaml:522]: > @mayankshah1607 we should bump the chart version, right?

No no, not for the main chart. We commit everything to `0.0.0` (`main` branch), and then bump the version at the time of releasing the chart (that is done by the CI action)

Comment (recharte) [reply charts/everest/crds/everest.yaml:522]: Sorry I commented without refreshing the page and didn't see your comments.

**Comment by mayankshah1607** [comment]
> it will become a part of 1.4.0..

Sorry folks, this is not really an issue, since the release pipeline will reset the CRDs back to the ones defined in the 1.4.0 branch.. https://github.com/percona/percona-helm-charts/blob/main/.github/workflows/everest-release.yaml#L33

No need to hold this, we can merge

**Comment by oksana-grishchenko** [comment]
> since the release pipeline will reset the CRDs back to the ones defined in the 1.4.0 branch

got a bit confused at first about reseting, but then realized how it happens, so let me share more details that might help others to understand. 
The crds will be generated from the freshly created `everest-opperator` _tag_ (let's say it's `v1.4.0-rc2`) which is based on the release branch and not going to contain whatever we merged to main after the release branch was created.

**Pull request merged**
